tests/test_utils.py

```diff
@@ -61,7 +61,7 @@ def test_get_extended_access_token():
 def test_get_extended_access_token_v23_plus():
     mock_request.return_value.status_code = 200
     mock_request.return_value.content = (
-        '{"access_token":"<extended access token>","token_type":"bearer"}'
+        '{"access_token":"<extended access token>","token_type":"bearer", "expires_in":5170982}'
```

Comment:
@jgorset The latest API version is 2.8, and it returns a key called `expires_in`.
```bash
curl -D - "https://graph.facebook.com/v2.8/oauth/access_token?client_id=xxx&grant_type=fb_exchange_token&client_secret=xxx&fb_exchange_token=xxx"

HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Pragma: no-cache
Cache-Control: private, no-cache, no-store, must-revalidate
facebook-api-version: v2.8
Expires: Sat, 01 Jan 2000 00:00:00 GMT
Content-Type: application/json; charset=UTF-8
x-fb-trace-id: CXWNY9XCDq/
x-fb-rev: 2728374
Vary: Accept-Encoding
X-FB-Debug: a/gcxxCFLErpXmls/SXNt0Dr0VchY3wik+ivDZ4CbhX6EvTaPEuBKcaj83gMQXCiE30g+BeUTDYglFXQ9Si3yA==
Date: Thu, 08 Dec 2016 14:44:53 GMT
Transfer-Encoding: chunked
Connection: keep-alive

{"access_token":"xxx","token_type":"bearer","expires_in":5103006}
```